src/test_workflow/benchmark_test/benchmark_args.py

```diff
@@ -50,117 +50,173 @@ class BenchmarkArgs:
-                            help="Enable Remote Store feature in OpenSearch")
-        parser.add_argument("--data-instance-type", dest="data_instance_type",
-                            help="EC2 instance type for data node, defaults to r5.xlarge.")
-        parser.add_argument("--workload", dest="workload", required=True,
-                            help="Name of the workload that OpenSearch Benchmark should run")
-        parser.add_argument("--benchmark-config", dest="benchmark_config",
-                            help="absolute filepath to custom benchmark.ini config")
-        parser.add_argument("--user-tag", dest="user_tag",
-                            help="Attach arbitrary text to the meta-data of each metric record")
-        parser.add_argument("--workload-params", dest="workload_params",
-                            help="With this parameter you can inject variables into workloads. Parameters differs "
-                                 "for each workload type. e.g., --workload-params \"number_of_replicas:1,number_of_shards:5\"")
-        parser.add_argument("--test-procedure", dest="test_procedure",
-                            help="Defines a test procedure to use. You can find a list of test procedures by using "
-                                 "opensearch-benchmark list test-procedures. E.g. --test-procedure=\"ingest-only\"")
-        parser.add_argument("--exclude-tasks", dest="exclude_tasks",
-                            help="Defines a comma-separated list of test procedure tasks not to run. E.g. --exclude-tasks=\"index-append\"")
-        parser.add_argument("--include-tasks", dest="include_tasks",
-                            help="Defines a comma-separated list of test procedure tasks to run. By default, all tasks listed in a test procedure array are run."
-                                 " E.g. --include-tasks=\"scroll\"")
-        parser.add_argument("--capture-node-stat", dest="telemetry", action="append_const", const="node-stats",
-                            help="Enable opensearch-benchmark to capture node stat metrics such as cpu, mem, jvm etc as well.")
-        parser.add_argument("--capture-segment-replication-stat", dest="telemetry", action="append_const",
-                            const="segment-replication-stats",
-                            help="Enable opensearch-benchmark to segment_replication stat metrics such as replication lag.")
-        parser.add_argument("--telemetry-params", dest="telemetry_params",
-                            help="Allows to set parameters for telemetry devices. Accepts json input.")
-        parser.add_argument("-v", "--verbose", help="Show more verbose output.", action="store_const", default=logging.INFO,
-                            const=logging.DEBUG, dest="logging_level")
+        parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle or compare two tests")
+        subparsers = parser.add_subparsers(dest="command", required=True)
```

Comment:
add `help="Please provide either 'execute-test' to run benchmark or 'compare' command to compare benchmark runs."` to `command`